src/System.Net.WebSockets.Client/src/System/Net/WebSockets/WinHttpWebSocket.cs

```diff
@@ -795,12 +795,7 @@ private void CancelAllOperations()
 
             if (_operation.TcsReceive != null)
             {
-                var exception = new WebSocketException(
-                    WebSocketError.InvalidState,
-                    SR.Format(
-                        SR.net_WebSockets_InvalidState_ClosedOrAborted,
-                        "System.Net.WebSockets.InternalClientWebSocket",
-                        "Aborted"));
+                var exception = new OperationCanceledException("Aborted");
 
                 _operation.TcsReceive.TrySetException(exception);
```

Comment:
Rather than construction an OperationCanceledException, you can just do:
```C#
_operation.TcsReceive.TrySetCanceled();
```
unless the "Aborted" string is really important (and if it is, should it be a resource string to enable localization)?